packages/backend/src/api/resources/SamlConnection.ts

```diff
@@ -0,0 +1,49 @@
+import type { AttributeMapping } from './AttributeMapping';
+import type { SamlConnectionJSON } from './JSON';
+
+export class SamlConnection {
```

Comment:
🔧 The following properties can be null `idpEntityId`, `idpSsoUrl`, `idpCertificate`, `idpMetadataUrl`

We also missing the optional `idpMetadata` property